be/src/exec/pipeline/operator.h

```diff
@@ -161,7 +161,12 @@ class Operator {
     std::shared_ptr<RuntimeProfile> _runtime_profile;
     std::shared_ptr<RuntimeProfile> _common_metrics;
     std::shared_ptr<RuntimeProfile> _unique_metrics;
-    MemTracker* _mem_tracker = nullptr;
+
+    // All the memory usage will be automatically added to the instance level MemTracker by memory allocate hook
+    // But for some special operators, we hope to see the memory usage of some special data structures,
+    // such as hash table of aggregate operators.
+    // So the following indenpendent MemTracker is introduced to record these memory usage
+    std::shared_ptr<MemTracker> _mem_tracker = nullptr;
```

Comment:
could we limit this indenpendent MemTracker memory usage?     the all operators memory usage  =  instance memtracker memory usage + this  indenpendent MemTracker memory usage?

Reply:
The purpose of  the extra indenpendent MemTracker is to show the memory usage of some specificed data structure. And it's memory is already recoreded into the instance's MemTracker by hook. So the duplicate record here is only for profile analysis.